### PR TITLE
Updated the root structure of Client side

### PR DESCRIPTION
**Pull request by veyselboybay**
Created routing structure using react-router-dom and added some links and updated the folder structure.

### CODE REVIEW

File: Client/src/App.jsx [hunk 1]
```diff
@@ -1,43 +1,23 @@
...
-      <div className="app-container" style={{ width: '500px' }}>
-      <DropDown
-        id="team-member-autocomplete"
-        options={teamMembers}
-        label="Select team member"
-        value={selectedTeamMember}
-        onChange={handleTeamMemberChange}
-      />
-    </div>
+      <Routes>
```
Comment (popnfresh234) [inline Client/src/App.jsx:13]: Nice to see the router in place

File: Client/src/main.jsx [hunk 2]
```diff
@@ -2,9 +2,12 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
 import './index.css'
+import { BrowserRouter as Router, HashRouter } from 'react-router-dom'
```
Comment (popnfresh234) [inline Client/src/main.jsx:5]: HashRouter unused at this time

**Review by popnfresh234 — APPROVED** [review]
Looks good to me!

File: Client/src/main.jsx (hunk 2 above)
Comment (veyselboybay) [reply Client/src/main.jsx:5]: The reason I put HashRouter is that route location is not detected when we use useLocation hook, I thought maybe we could switch it later for a solution if we encounter an issue.

Comment (popnfresh234) [reply Client/src/main.jsx:5]: Got it, thanks for the explanation :+1:

Comment (veyselboybay) [reply Client/src/main.jsx:5]: Thanks for the review 👍

**Pull request merged**